### ISSUE TITLE
I can't add a 6th member to my party for some reason

### ISSUE THREAD

**Issue by thegamingnobody** [domtronn/nuzlocke.app#128]
**Describe the bug**

In my party I have 5 out of 6 pokemon selected. When going through my box and such, the option to add a pokemon to my party is nowhere to be found. When I click on the empty slot, it says to select max 1 pokemon to add to the party. However when I do this, the pop up window dissapears without doing anything. I suspect my save somehow thinks my party is already full.

I am using the site on the windows version of Opera GX, in case this matters.

**Comment by domtronn** [comment]
Hey @thegamingnobody I suspect that there's a pokemon in that slot that shouldn't be there... 

If you could send me the save file or the share code from [https://nuzlocke.app/saves](https://nuzlocke.app/saves)  I can take a look at what caused it.

But if you clear your team _(using the cross next to the Mini-Team)_ then re add them, it should work.

**Comment by thegamingnobody** [comment]
Clearing the party and adding them manually did indeed work for me, thanks!

**Comment by domtronn** [comment]
Awesome - When marking a Pokémon as dead, it _should_ remove it from your team... but I think you're right that there's some circumstance in which it stays in your team. I'll do some investigation and leave this issue open for any updates.

I had someone report something similar on discord, so thought I'd fixed it but will double check.

**Comment by chrisasma103** [comment]
Yeah, I had this exact same issue. Clearing the party seemed to solve it for me too.
